### PR TITLE
Add recipe for HmnTrimmer

### PR DESCRIPTION
**Pull request by guillaume-gricourt**
The following PR is adding the bioconda recipe for a tool called [HmnTrimmer](https://github.com/guillaume-gricourt/HmnTrimmer)

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by guillaume-gricourt** [comment]
@BiocondaBot please add label

File: recipes/hmntrimmer/conda_build_config.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+cxx_compiler_version:  # [unix]
```
Comment (daler) [inline recipes/hmntrimmer/conda_build_config.yaml:1]: Can you add a comment here on the rationale on the version override? That will help with long-term maintenance when this is updated in the future.

File: recipes/hmntrimmer/run_test.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+echo -e "\n\n*** TEST ***\n\n"
+
+HmnTrimmer 2> /dev/null || [[ "$?" == 1 ]]
+make test
```
Comment (daler) [inline recipes/hmntrimmer/run_test.sh:6]: Typically we [don't recommend running unit tests](https://bioconda.github.io/contributor/guidelines.html#tests) due to the risk of timing out CI infrastructure, but it seems to be fine here. Just FYI.

**Review by daler — CHANGES_REQUESTED** [review]

File: recipes/hmntrimmer/conda_build_config.yaml (hunk 1 above)
Comment (guillaume-gricourt) [reply recipes/hmntrimmer/conda_build_config.yaml]: Yes, it's added ccaa5a9

File: recipes/hmntrimmer/run_test.sh (hunk 2 above)
Comment (guillaume-gricourt) [reply recipes/hmntrimmer/run_test.sh:6]: Got it

**Review by bgruening — APPROVED** [review]

**Review by daler — APPROVED** [review]

**Comment by daler** [comment]
@BiocondaBot please merge

**Pull request merged**